README.md

```diff
@@ -1,2 +1,182 @@
-# enumerable
-In this project we will learn how to use a module inside a class. For this we will create a class MyList and a module MyEnumerable. Our module MyEnumerable will implement a subset of the functionality of Enumerable.
+# Create-Enumerable
+
+ Program based on Ruby to use modules and classes
+
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Create Enumerable <a name="about-project"></a>
+
+**Create Enumerable** is a program to check certain conditions on a list. 
+
+- Check all. 
+- Check any.
+- Filter
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>N/A</li>
+
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Check all
+- Check any
+- Filter
+
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- LIVE DEMO -->
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Getting Started -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To run the project locally, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+
+- Install [Git](https://git-scm.com/)
+- Install [Node](https://nodejs.org/en/)
+- Install a code editor, I suggest [VsCode](https://code.visualstudio.com/)
+
+### Setup
+
+Clone the repo into your local machine:
+
+```bash
+git clone git@github.com:edgardsoza/enumerable/tree/enumerable.git
+```
+
+Install the dependencies using npm:
+```bash
+npm install
+```
+
+### Usage
+
+To start the development server run
+```bash
+npm run start
+```
+- this command will start the development server on `localhost:8080` address
+
+To create and optimized production build run
+```bash
+npm run build
+```
+- this command will create a bunduled chunks of code inside `dist/` folder
+
+
+<p align="right"><a href="#readme-top">👆</a></p>
```

Comment:
- [ ] Kindly consider adding correct information on how to run your project locally and remember, 🤔 this is not javascript but ruby, you should add something similar to `ruby my_list.rb` instead. ✍🏻